### PR TITLE
Optimise assert, fix webpanel debug, fix gamegravity default, fix TimeHint

### PR DESCRIPTION
**Pull request by ccuser44**
- Optimise `assert` calls
- ~~Remove legacy givepoints command, [PointsService has been disabled totally](https://devforum.roblox.com/t/retiring-pointsservice/2318767)~~
- Fix gamegravity command not being able to reset gravity
- Fix TimeHint from not working at all
- Fix webpanel not understanding error codes properly

### CODE REVIEW

**Comment by Dimenpsyonal** [comment]
No more givepoints command? No.

**Comment by ccuser44** [comment]
> No more givepoints command? No.

Not needed and now does literally nothing because PointService does not work at all

**Comment by Dimenpsyonal** [comment]
I use it for the client notification

**Comment by ccuser44** [comment]
> I use it for the client notification

`:rbxnotify player Points awarded. You have been awarded X points`

**Comment by Dimenpsyonal** [comment]
Too bad

**Comment by ccuser44** [comment]
> Too bad

Bruh

**Comment by ccuser44** [comment]
`[":giveppoints <player> <points>"] = ":script game:GetService("PointsService"):AwardPoints(game:GetService:("Players"):FindFirstChild("<player>"), <points>)"`

**Comment by Dimenpsyonal** [comment]
What

**Comment by ccuser44** [comment]
> What

Its an alias

File: MainModule/Server/Commands/Fun.lua [hunk 1]
```diff
@@ -4773,8 +4773,7 @@ return function(Vargs, env)
 			Fun = true;
 			AdminLevel = "Admins";
 			Function = function(plr: Player, args: {string})
-				local num = assert(tonumber(args[1]), "Missing gravity value (or enter 'fix' to reset to normal)'")
-				workspace.Gravity = num or 196.2
+				workspace.Gravity = args[1] == "fix" and 196.2 or assert(tonumber(args[1]), "Missing gravity value (or enter 'fix' to reset to normal)'")
```
Comment (fxeP1) [inline MainModule/Server/Commands/Fun.lua:4776]: I suppose a change to "default" would be more suitable as _some_ games do change their default `workspace.Gravity` property or allow the "fix" argument to fetch the old Gravity and actually "fix" it (i.e: caching old  before changing gravity, etc)

Comment (ccuser44) [reply MainModule/Server/Commands/Fun.lua]: Done

**Review by fxeP1 — APPROVED** [review]
LGTM

**Comment by Dimenpsyonal** [comment]
DLGTM

**Comment by ccuser44** [comment]
> DLGTM

```diff
-DLGTM
+LGTM
```

**Comment by ccuser44** [comment]
@Dimenpsyonal you forgot to merge this as well

**Comment by Dimenpsyonal** [comment]
I will not be merging this.

**Comment by Dimenpsyonal** [comment]
Get some other noob to merge this

**Comment by ccuser44** [comment]
> I will not be merging this.

Why?

**Comment by WalkerOfBacon** [comment]
bro has a grudge against this pr

**Comment by Dimenpsyonal** [comment]
Removes givepoints command

**Comment by WalkerOfBacon** [comment]
you will merge it NOW

**Comment by ccuser44** [comment]
> Removes givepoints command

Ok givepoints removal will be seperated to a different pull

**Comment by WalkerOfBacon** [comment]
real

**Pull request merged**